hw/ip/aes/doc/dv_plan/index.md

```diff
@@ -59,26 +59,30 @@ The `aes_base_vseq` virtual sequence is extended from `cip_base_vseq` and serves
 All test sequences are extended from `aes_base_vseq`.
 It provides commonly used handles, variables, functions and tasks that the test sequences can simple use / call.
 Some of the most commonly used tasks / functions are as follows:
-* aes_init:    Configure aes control and can via a knob be set to randomize register content before starting a test.
+* aes_init:    Initialize the AES module from the randomized environment variables in the config.
 * set_op:      Set AES operation to encrypt or decrypt.
-* write_key:   Write initial key to aes init key registers.
+* write_key:   Write initial key to AES init key registers.
 * add_data:    Add the next 128 block to the input registers.
-* read_output:  Poll the status reg for dataready bit and read the result from aes output regs
+* read_output:  Poll the status reg for dataready bit and read the result from AES output regs
+* clear_reg:  Based on the input this function clears data input, data output or key registers or any combination of these.
+* set_manual_trigger: Chooses between AES auto start and manual start.
+* trigger_start:  Set the start bit to trigger a new encryption/decryption.
 
 #### Functional coverage
 To ensure high quality constrained random stimulus, it is necessary to develop a functional coverage model.
-The following covergroups have been developed to prove that the test intent has been adequately met:
+The following cover groups have been developed to prove that the test intent has been adequately met:
 *---  Work in Progress ---
```

Comment:
This "comment" messes up the rendering both on Hugo and just here (everything below is italic). Please fix this.

Reply:
fixed

Reply:
Thx.